### PR TITLE
Extra folder housekeeping

### PR DESCRIPTION
**Pull request by kubamarchwicki**
As the prerequisite to scrapping Groovy from the project (and moving it to a sparate project) I did some housekeeping in the extra/ folder. That inncluded:

* Proper naming in project description - as in the main project
* Ignoring a test that probably never worked

### CODE REVIEW

File: extra/nosql/hibernate-ogm/src/test/java/org/javaee7/extra/nosql/hibernateogm/EhcacheTest.java [hunk 1]
```diff
@@ -4,12 +4,14 @@
...
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Ignore;
 import org.junit.runner.RunWith;
 
 /**
  * @author Guillaume Scheibel <guillaume.scheibel@gmail.com>
  */
 @RunWith(Arquillian.class)
+@Ignore("This test is probably a sad panda; never been successful sad panda.")
```
Comment (bartoszmajsak) [inline extra/nosql/hibernate-ogm/src/test/java/org/javaee7/extra/nosql/hibernateogm/EhcacheTest.java]: What's the point of keeping it then?

File: extra/twitter-search/pom.xml [hunk 2]
```diff
@@ -12,12 +12,18 @@
     <artifactId>twitter-search</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>war</packaging>
+    <name>Java EE 7 Samples: extras - twitter-search</name>
 
     <dependencies>
+        <!--
+            Non Maven dependency: https://github.com/arun-gupta/twitter-api
```
Comment (bartoszmajsak) [inline extra/twitter-search/pom.xml]: This could be mavenized in github itself :)

**Comment by bartoszmajsak** [comment]
:+1:  if only Travis accepts we can merge it. Thanks Kuna!

File: extra/twitter-search/pom.xml (hunk 2 above)
Comment (kubamarchwicki) [inline extra/twitter-search/pom.xml]: :+1: forgot about this little tweak! - fixed

**Comment by arun-gupta** [comment]
No response from @kubamarchwicki  and so closing the PR.

**Comment by kubamarchwicki** [comment]
@arun-gupta if this work on your machine can you please provide additional README documentation that nosql datastore is required and how to setup. Otherwise, the example is pretty meaningless for a novice user. 

Additionally, the commit  08eb6a4 was including some housekeeping in twitter search dependencies. Those could have been merged - otherwise the build is failing. The https://github.com/arun-gupta/twitter-api is a non-maven dependency and have to be provided either through system path of through jitpack.

**Comment by arun-gupta** [comment]
Lets remove the project from the build and it can be included back when the instructions are updated. Removing the files does not seem right.

**Pull request closed without merging**